packages/salesforcedx-vscode-core/src/commands/templates/forceFunctionCreate.ts

```diff
@@ -88,6 +94,24 @@ export class ForceFunctionCreateExecutor extends BaseTemplateCommand {
       });
     }
   }
+
+  public logMetric(
+    logName: string | undefined,
+    hrstart: [number, number],
+    properties?: Properties,
+    measurements?: Measurements
+  ) {
+    if (properties) {
+      const fileExtension = this.getFileExtension().replace('.', '');
```

Comment:
can we compare with `.js` and `.java` instead of replacing the `.`?

Reply:
Sure can!